cli/azd/test/functional/up_test.go

```diff
@@ -241,7 +242,7 @@ func Test_CLI_Up_Down_ContainerApp(t *testing.T) {
 			cli.WorkingDirectory = dir
 			cli.Env = append(os.Environ(),
 				"AZURE_LOCATION=eastus2",
-				fmt.Sprintf("AZURE_PROVISION_CONTAINER_APP=%t", tt.provisionContainerApp))
+				fmt.Sprintf("AZURE_PROVISION_CONTAINER_APP=%t", tc.provisionContainerApp))
```

Comment:
A linter complained locally about the fact that the closure captured the loop variable reference instead of the value.  This is a common footgun with parallel tests (here's hoping they fix this in the language sometime soon).  I fixed it here.

Reply:
Good catch. Will have to remind myself to be more careful about for-loop parallels